### DISCUSSION TITLE
It's alive!

### DISCUSSION THREAD

**Discussion by JackySw** [Tichau/FileConverter#385] [Q&A]
After all these years, version 2.0 just dropped! Very awesome to see this project came to life again.

I would say, hit the Donate button on his website: https://file-converter.io/ to show some appreciation for the hard work and to make new updates possible.

**Comment by Tichau** [comment]
Thanks !!

It was a lot of work to finish this big step, but I'm happy it's out.

**Comment by glecas** [comment]
Yes!!!!! Woohoo! This is one of my must-have apps and I am so happy to be able to convert _.heic_ files now finally. I can't wait to play around with the new features and capabilities. 🥳🎉

And yes, I just donated five seconds ago! 💸

**Reply by Tichau** [reply]
Thanks a lot ! for reaching out and for the donation
